### PR TITLE
Huong/scripts

### PR DESCRIPTION
**Pull request by huonglarne**
Fixed Thang's scripts:
- Use moreh-switch device instead of moreh-switch model command because it enables multiple devices on the same machine
- Change order of some arguments

Training scripts for:
- speech recognition
- semantic segmentation
- summarization
- language modeling

### CODE REVIEW

File: examples/pytorch/semantic-segmentation/run_semantic_segmentation.py [hunk 1]
```diff
@@ -388,6 +388,7 @@ def compute_metrics(eval_pred):
         cache_dir=model_args.cache_dir,
         revision=model_args.model_revision,
         use_auth_token=True if model_args.use_auth_token else None,
+        ignore_mismatched_sizes=True,
```
Comment (yeomni) [inline examples/pytorch/semantic-segmentation/run_semantic_segmentation.py:391]: Does it cause RuntimeError if it's False?

Comment (huonglarne) [reply examples/pytorch/semantic-segmentation/run_semantic_segmentation.py:391]: oh sorry, I mistunderstood your question. I never tried it with False, the default command use True

File: examples/pytorch/language-modeling/model_batchsize.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+gpt2 8
```
Comment (yeomni) [inline examples/pytorch/language-modeling/model_batchsize.txt:1]: Is GPT2 the only available model here in language-modeling? Or are there more to be added later?

File: examples/pytorch/speech-recognition/train_seq2seq.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+device_id=$3
+log_file="${LOG_DIR}/${model_name}.log"
+output_dir="$OUTPUT_DIR/$model_name"
+
+## END OF SETTINGS ##
+
+export TRANSFORMERS_CACHE=/nas/huggingface_pretrained_models
+export HF_DATASETS_CACHE=/nas/common_data/huggingface
+
+args="
```
Comment (yeomni) [inline examples/pytorch/speech-recognition/train_seq2seq.sh:18]: I think it's better to have a separated `.yml` for this, as well as for all other scripts.

Comment (huonglarne) [reply examples/pytorch/speech-recognition/train_seq2seq.sh:18]: @yeomni I noted your comments. I will fix them in the next pull requests

**Pull request merged**